### PR TITLE
Fix more pronunciations (1/N)

### PR DESCRIPTION
**Pull request by xatier**
This is 1/N break up of #504, we fix the incorrect pronunciations in this commit first.

### CODE REVIEW

**Comment by xatier** [comment]
@ChiahongHong I tried to follow the pronunciations from the dictionaries, if you noticed some "commonly misused" ones, please kindly let me know and I'll add them back.

File: Source/Data/BPMFMappings.txt [hunk 1]
```diff
@@ -132741,7 +132736,7 @@
 音樂系 ㄧㄣ ㄩㄝˋ ㄒㄧˋ
 音樂總監 ㄧㄣ ㄩㄝˋ ㄗㄨㄥˇ ㄐㄧㄢ
 音樂老師 ㄧㄣ ㄩㄝˋ ㄌㄠˇ ㄕ
-音樂課 ㄧㄣ ㄌㄜˋ ㄎㄜˋ
+音樂課 ㄧㄣ ㄩㄝˋ ㄎㄜˋ
```
Comment (ChiahongHong) [inline Source/Data/BPMFMappings.txt]: Conflict: https://github.com/openvanilla/McBopomofo/pull/505

Comment (xatier) [reply Source/Data/BPMFMappings.txt]: removed them to avoid conflicts

**Comment by ChiahongHong** [comment]
Thank you!

Found a new incorrect pronunciation; it can be corrected as well

- [出使](https://dict.concised.moe.edu.tw/dictView.jsp?ID=32530) `ㄔㄨ ㄕˋ` → `ㄔㄨ ㄕˇ`

---

`煽` 教育部各字典皆未收 `ㄕㄢˋ` 這個音，不確定會不會有人這樣讀，如果是就先不處理留作記錄

- [煽動](https://dict.concised.moe.edu.tw/dictView.jsp?ID=34479)力 `ㄕㄢˋ ㄉㄨㄥˋ` → `ㄕㄢ ㄉㄨㄥˋ`
- 煽動性
- [煽情](https://dict.concised.moe.edu.tw/dictView.jsp?ID=34481) `ㄕㄢˋ ㄑㄧㄥˊ` → `ㄕㄢ ㄑㄧㄥˊ`
- 煽情主義
- [煽惑](https://dict.concised.moe.edu.tw/dictView.jsp?ID=34480) `ㄕㄢˋ ㄏㄨㄛˋ` → `ㄕㄢ ㄏㄨㄛˋ`

**Comment by xatier** [comment]
Fixed `出使 ㄔㄨ ㄕˋ → ㄔㄨ ㄕˇ`.

Yes, I didn't find any references for `煽 ㄕㄢˋ`, I made them all consistent as `ㄕㄢ`

```
$ ./find.py 煽 ㄕㄢ
煽動 ㄕㄢ ㄉㄨㄥˋ
煽動力 ㄕㄢ ㄉㄨㄥˋ ㄌㄧˋ
煽動性 ㄕㄢ ㄉㄨㄥˋ ㄒㄧㄥˋ
煽客 ㄕㄢ ㄎㄜˋ
煽情 ㄕㄢ ㄑㄧㄥˊ
煽情主義 ㄕㄢ ㄑㄧㄥˊ ㄓㄨˇ ㄧˋ
煽惑 ㄕㄢ ㄏㄨㄛˋ
煽火 ㄕㄢ ㄏㄨㄛˇ
煽色腥 ㄕㄢ ㄙㄜˋ ㄒㄧㄥ
煽起 ㄕㄢ ㄑㄧˇ
煽起來 ㄕㄢ ㄑㄧˇ ㄌㄞˊ
煽風點火 ㄕㄢ ㄈㄥ ㄉㄧㄢˇ ㄏㄨㄛˇ
```

File: Source/Data/BPMFMappings.txt [hunk 2]
```diff
@@ -135971,8 +135966,8 @@
 高射 ㄍㄠ ㄕㄜˋ
 高射炮 ㄍㄠ ㄕㄜˋ ㄆㄠˋ
 高射砲 ㄍㄠ ㄕㄜˋ ㄆㄠˋ
-高射砲彈 ㄍㄠ ㄧㄝˋ ㄆㄠˋ ㄉㄢˋ
-高射部隊 ㄍㄠ ㄧㄝˋ ㄅㄨˋ ㄉㄨㄟˋ
+高射砲彈 ㄍㄠ  ㄕㄜˋㄆㄠˋ ㄉㄢˋ
+高射部隊 ㄍㄠ  ㄕㄜˋㄅㄨˋ ㄉㄨㄟˋ
```
Comment (ChiahongHong) [inline Source/Data/BPMFMappings.txt]: - `ㄍㄠ  ㄕㄜˋㄆㄠˋ ㄉㄢˋ` → `ㄍㄠ ㄕㄜˋ ㄆㄠˋ ㄉㄢˋ`
- `ㄍㄠ  ㄕㄜˋㄅㄨˋ ㄉㄨㄟˋ` → `ㄍㄠ ㄕㄜˋ ㄅㄨˋ ㄉㄨㄟˋ`

Comment (xatier) [reply Source/Data/BPMFMappings.txt]: Oops, thanks!

File: Source/Data/BPMFMappings.txt [hunk 3]
```diff
@@ -51343,7 +51342,6 @@
 怯怯的 ㄑㄩㄝˋ ㄑㄩㄝˋ ㄉㄜ˙
 怯意 ㄑㄧㄝˋ ㄧˋ
 怯意 ㄑㄩㄝˋ ㄧˋ
-怯懦 ㄑㄧㄝˋ ㄋㄨㄛˋ
```
Comment (ChiahongHong) [inline Source/Data/BPMFMappings.txt]: Removing this line might cause inconsistency with the context. We may choose to add `ㄑㄧㄝˋ` for `怯場`, `怯懦的`, `怯聲怯氣`, or remove the `ㄑㄧㄝˋ` altogether.

**Comment by ChiahongHong** [comment]
@xatier I have double-checked all these changes. Thank you! 🎉

File: Source/Data/BPMFMappings.txt (hunk 3 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: I've reviewed all `怯` (`vimdiff <(./find.py 怯 ㄑㄧㄝˋ) <(./find.py 怯 ㄑㄩㄝˋ)`) and believe they are highly inconsistent, my decision is to add both for fault tolerance in this case.


```diff
diff --git a/Source/Data/BPMFMappings.txt b/Source/Data/BPMFMappings.txt
index d0ef104..24d6246 100644
--- a/Source/Data/BPMFMappings.txt
+++ b/Source/Data/BPMFMappings.txt
@@ -50311,6 +50311,7 @@
 心藥 ㄒㄧㄣ ㄧㄠˋ
 心虛 ㄒㄧㄣ ㄒㄩ
 心虛膽怯 ㄒㄧㄣ ㄒㄩ ㄉㄢˇ ㄑㄧㄝˋ
+心虛膽怯 ㄒㄧㄣ ㄒㄩ ㄉㄢˇ ㄑㄩㄝˋ
 心血 ㄒㄧㄣ ㄒㄧㄝˇ
 心血來潮 ㄒㄧㄣ ㄒㄧㄝˇ ㄌㄞˊ ㄔㄠˊ
 心血白費 ㄒㄧㄣ ㄒㄧㄝˇ ㄅㄞˊ ㄈㄟˋ
@@ -51333,6 +51334,7 @@
 怪過 ㄍㄨㄞˋ ㄍㄨㄛˋ
 怪雨 ㄍㄨㄞˋ ㄩˇ
 怪魚 ㄍㄨㄞˋ ㄩˊ
+怯場 ㄑㄧㄝˋ ㄔㄤˇ
 怯場 ㄑㄩㄝˋ ㄔㄤˇ
 怯弱 ㄑㄧㄝˋ ㄖㄨㄛˋ
 怯弱 ㄑㄩㄝˋ ㄖㄨㄛˋ
@@ -51342,7 +51344,9 @@
 怯怯的 ㄑㄩㄝˋ ㄑㄩㄝˋ ㄉㄜ˙
 怯意 ㄑㄧㄝˋ ㄧˋ
 怯意 ㄑㄩㄝˋ ㄧˋ
+怯懦 ㄑㄧㄝˋ ㄋㄨㄛˋ
 怯懦 ㄑㄩㄝˋ ㄋㄨㄛˋ
+怯懦的 ㄑㄧㄝˋ ㄋㄨㄛˋ ㄉㄜ˙
 怯懦的 ㄑㄩㄝˋ ㄋㄨㄛˋ ㄉㄜ˙
 怯生生 ㄑㄧㄝˋ ㄕㄥ ㄕㄥ
 怯生生 ㄑㄩㄝˋ ㄕㄥ ㄕㄥ
@@ -51350,6 +51354,7 @@
 怯生生的 ㄑㄩㄝˋ ㄕㄥ ㄕㄥ ㄉㄜ˙
 怯疑 ㄑㄧㄝˋ ㄧˊ
 怯疑 ㄑㄩㄝˋ ㄧˊ
+怯聲怯氣 ㄑㄧㄝˋ ㄕㄥ ㄑㄧㄝˋ ㄑㄧˋ
 怯聲怯氣 ㄑㄩㄝˋ ㄕㄥ ㄑㄩㄝˋ ㄑㄧˋ
 怯頭怯腦 ㄑㄧㄝˋ ㄊㄡˊ ㄑㄧㄝˋ ㄋㄠˇ
 怯頭怯腦 ㄑㄩㄝˋ ㄊㄡˊ ㄑㄩㄝˋ ㄋㄠˇ
@@ -53245,6 +53250,7 @@
 懦夫 ㄋㄨㄛˋ ㄈㄨ
 懦弱 ㄋㄨㄛˋ ㄖㄨㄛˋ
 懦怯 ㄋㄨㄛˋ ㄑㄧㄝˋ
+懦怯 ㄋㄨㄛˋ ㄑㄩㄝˋ
 懲一警百 ㄔㄥˊ ㄧ ㄐㄧㄥˇ ㄅㄞˊ
 懲前毖後 ㄔㄥˊ ㄑㄧㄢˊ ㄅㄧˋ ㄏㄡˋ
 懲惡勸善 ㄔㄥˊ ㄜˋ ㄑㄩㄢˋ ㄕㄢˋ
@@ -87425,6 +87431,7 @@
 生怕 ㄕㄥ ㄆㄚˋ
 生性 ㄕㄥ ㄒㄧㄥˋ
 生性固執 ㄕㄥ ㄒㄧㄥˋ ㄍㄨˋ ㄓˊ
+生怯怯 ㄕㄥ ㄑㄧㄝˋ ㄑㄧㄝˋ
 生怯怯 ㄕㄥ ㄑㄩㄝˋ ㄑㄩㄝˋ
 生恐 ㄕㄥ ㄎㄨㄥˇ
 生息 ㄕㄥ ㄒㄧˊ
@@ -89188,6 +89195,7 @@
 瘦成皮包骨 ㄕㄡˋ ㄔㄥˊ ㄆㄧˊ ㄅㄠ ㄍㄨˇ
 瘦瘦 ㄕㄡˋ ㄕㄡˋ
 瘦瘦小小 ㄕㄡˋ ㄕㄡˋ ㄒㄧㄠˇ ㄒㄧㄠˇ
+瘦瘦怯怯 ㄕㄡˋ ㄕㄡˋ ㄑㄧㄝˋ ㄑㄧㄝˋ
 瘦瘦怯怯 ㄕㄡˋ ㄕㄡˋ ㄑㄩㄝˋ ㄑㄩㄝˋ
 瘦瘦長長 ㄕㄡˋ ㄕㄡˋ ㄔㄤˊ ㄔㄤˊ
 瘦瘦高高 ㄕㄡˋ ㄕㄡˋ ㄍㄠ ㄍㄠ
@@ -104874,6 +104882,7 @@
 膽小 ㄉㄢˇ ㄒㄧㄠˇ
 膽小如鼠 ㄉㄢˇ ㄒㄧㄠˇ ㄖㄨˊ ㄕㄨˇ
 膽小鬼 ㄉㄢˇ ㄒㄧㄠˇ ㄍㄨㄟˇ
+膽怯 ㄉㄢˇ ㄑㄧㄝˋ
 膽怯 ㄉㄢˇ ㄑㄩㄝˋ
 膽戰心驚 ㄉㄢˇ ㄓㄢˋ ㄒㄧㄣ ㄐㄧㄥ
 膽敢 ㄉㄢˇ ㄍㄢˇ
```

**Comment by xatier** [comment]
@lukhnos this one should be ready to go, after this PR, I'll submit subsequent PRs for characters in #504 individually (fix one character at a time).

**Comment by xatier** [comment]
Rebase on top of master.

**Comment by xatier** [comment]
Rebase on top of master.

File: Source/Data/BPMFMappings.txt [hunk 4]
```diff
@@ -30491,6 +30492,7 @@
 喧鬧不休 ㄒㄩㄢ ㄋㄠˋ ㄅㄨˋ ㄒㄧㄡ
 喪主 ㄙㄤ ㄓㄨˇ
 喪主 ㄙㄤˋ ㄓㄨˇ
+喪亂 ㄙㄤ ㄌㄨㄢˋ
 喪亂 ㄙㄤˋ ㄌㄨㄢˋ
```
Comment (tianjianjiang) [inline Source/Data/BPMFMappings.txt:30496]: 看來這 PR 移除了某些俗音和錯音，為求一致，要不要把這一條也刪掉？

File: Source/Data/BPMFMappings.txt [hunk 5]
```diff
@@ -83854,7 +83862,7 @@
 熟透 ㄕㄡˊ ㄊㄡˋ
 熟鐵 ㄕㄨˊ ㄊㄧㄝˇ
 熟面孔 ㄕㄡˊ ㄇㄧㄢˋ ㄎㄨㄥˇ
-熟食 ㄕㄨˊ ㄕˊ
+熟食 ㄕㄡˊ ㄕˊ
```
Comment (tianjianjiang) [inline Source/Data/BPMFMappings.txt]: 從前面的其他變動看來，或許一致的方向是保留甚至新增更多「ㄕㄨˊ 」？

File: Source/Data/BPMFMappings.txt [hunk 6]
```diff
@@ -104017,11 +104026,12 @@
...
 胃鏡室 ㄨㄟˋ ㄐㄧㄥˋ ㄕˋ
 背上 ㄅㄟ ㄕㄤˋ
+背上 ㄅㄟˋ ㄕㄤˋ
 背上來 ㄅㄟ ㄕㄤˋ ㄌㄞˊ
 背上去 ㄅㄟ ㄕㄤˋ ㄑㄩˋ
 背上有 ㄅㄟˋ ㄕㄤˋ ㄧㄡˇ
-背下來 ㄅㄟˋ ㄒㄧㄚˋ ㄌㄞˊ
-背不了 ㄅㄟˋ ㄅㄨˋ ㄌㄜ˙
+背下來 ㄅㄟ ㄒㄧㄚˋ ㄌㄞˊ
+背不了 ㄅㄟ ㄅㄨˋ ㄌㄜ˙
```
Comment (tianjianjiang) [inline Source/Data/BPMFMappings.txt]: 兩種發音似乎都通？破音異義的樣子。

File: Source/Data/BPMFMappings.txt [hunk 7]
```diff
@@ -115787,7 +115795,7 @@
 譬如 ㄆㄧˋ ㄖㄨˊ
 譬如說 ㄆㄧˋ ㄖㄨˊ ㄕㄨㄛ
 譬若 ㄆㄧˋ ㄖㄨㄛˋ
-譯傳 ㄧˋ ㄓㄨㄢˋ
+譯傳 ㄧˋ ㄔㄨㄢˊ
```
Comment (tianjianjiang) [inline Source/Data/BPMFMappings.txt]: 這可能是某些佛教典籍的翻譯傳記。
ㄧˋ ㄔㄨㄢˊ 大概是「驛」傳。

File: Source/Data/BPMFMappings.txt [hunk 8]
```diff
@@ -119247,7 +119253,7 @@
 身分證 ㄕㄣ ㄈㄣ ㄓㄥˋ
```
Comment (tianjianjiang) [inline Source/Data/BPMFMappings.txt]: 照底下「身分辦識」的方向來看，這則可能也要刪掉？

File: Source/Data/BPMFMappings.txt [hunk 9]
```diff
@@ -123810,11 +123816,10 @@
 遺孤 ㄧˊ ㄍㄨ
 遺孽 ㄧˊ ㄋㄧㄝˋ
 遺容 ㄧˊ ㄖㄨㄥˊ
-遺屬 ㄨㄟˋ ㄕㄨˇ
```
Comment (tianjianjiang) [inline Source/Data/BPMFMappings.txt:123813]: 正確發音的詞條似乎不在檔案裡？

File: Source/Data/phrase.occ [hunk 10]
```diff
@@ -131010,7 +131010,6 @@
 遺孤 7
 遺孽 1
 遺容 2
-遺屬 6
```
Comment (tianjianjiang) [inline Source/Data/phrase.occ]: 也許不該刪掉？

**Review by tianjianjiang — CHANGES_REQUESTED** [review]

File: Source/Data/BPMFMappings.txt (hunk 7 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: https://dict.revised.moe.edu.tw/dictView.jsp?ID=151773&q=1&word=%E8%AD%AF%E5%82%B3

File: Source/Data/BPMFMappings.txt (hunk 9 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt:123813]:
```
Source/Data/phrase.occ
31616:國父遺囑 0
131004:遺囑 17

Source/Data/BPMFMappings.txt
32270:國父遺囑 ㄍㄨㄛˊ ㄈㄨˋ ㄧˊ ㄓㄨˇ
123810:遺囑 ㄧˊ ㄓㄨˇ
```

File: Source/Data/BPMFMappings.txt (hunk 4 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt:30496]: This is the correct pronunciation.

https://www.moedict.tw/%E5%96%AA%E4%BA%82

File: Source/Data/BPMFMappings.txt (hunk 5 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: Will do in subsequent PRs.

File: Source/Data/BPMFMappings.txt (hunk 9 above)
Comment (ChiahongHong) [reply Source/Data/BPMFMappings.txt:123813]: @xatier 後來才發現 `遺屬` 不是 `遺囑` 的錯字，是指保險的那個 `遺屬`，例如「遺屬年金」

所以可能要加回來，改讀音~

File: Source/Data/BPMFMappings.txt (hunk 6 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: Good catch, adding them back.

File: Source/Data/BPMFMappings.txt (hunk 8 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: I would prefer to keep this, as this is a commonly misused one.

File: Source/Data/BPMFMappings.txt (hunk 7 above)
Comment (tianjianjiang) [reply Source/Data/BPMFMappings.txt]: yeah, that's "驛"

File: Source/Data/BPMFMappings.txt (hunk 4 above)
Comment (tianjianjiang) [reply Source/Data/BPMFMappings.txt:30496]: @xatier 我是指「ㄙㄤˋ」那一條

File: Source/Data/BPMFMappings.txt (hunk 7 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: Oh oops, I must be too tired last night.

File: Source/Data/BPMFMappings.txt (hunk 8 above)
Comment (tianjianjiang) [reply Source/Data/BPMFMappings.txt]: @xatier 如果是這樣，那或許就別刪「身分辨識」？
我通常會建議盡量不要刪本來就能用的東西。

Comment (xatier) [reply Source/Data/BPMFMappings.txt]: I've never head people saying `身分辨識 ㄕㄣ ㄈㄣ ㄅㄧㄢˋ ㄕˋ`, but `身分證 ㄕㄣ ㄈㄣ ㄓㄥˋ` seems quite common. Anyway, I've added it back.

**Review by tianjianjiang — APPROVED** [review]

**Comment by xatier** [comment]
Rebase on top of master.

**Comment by xatier** [comment]
@lukhnos thank you for merging this!

**Comment by xatier** [comment]
@lukhnos would you mind having a sync to the Linux repo? So that I can enjoy the revisited dictionary on my box.

Similar to https://github.com/openvanilla/fcitx5-mcbopomofo/pull/143

**Pull request merged**